brayns/PluginManager.cpp

```diff
@@ -43,6 +40,11 @@ typedef ExtensionPlugin* (*CreateFuncType)(int, const char**);
 
 PluginManager::PluginManager(int argc, const char** argv)
 {
+    bool help = false;
+    for (int i = 0; i < argc && !help; ++i)
+        if (std::string(argv[i]) == "--help")
+            help = true;
```

Comment:
const bool help = std::count_if(argv, argv + argc, [](const char* arg) {
        return std::strcmp(*arg, "--help") == 0;
    }) > 0;

Reply:
lol, shame on the original author of this in ParametersManager.cpp :)